edx_proctoring/tests/test_views.py

```diff
@@ -1107,7 +1107,8 @@ def test_download_software_clicked_action(self):
 
     @ddt.data(
         ('submit', ProctoredExamStudentAttemptStatus.submitted),
-        ('decline', ProctoredExamStudentAttemptStatus.declined)
+        ('decline', ProctoredExamStudentAttemptStatus.declined),
+        ('error', ProctoredExamStudentAttemptStatus.error)
```

Comment:
In Python files (at least) you can use trailing commas to keep future diffs cleaner.